playbooks/hc-ansible-deployment/he_gluster_vars.json

```diff
@@ -11,5 +11,6 @@
   "he_storage_domain_path": "/engine",
   "he_storage_domain_addr": "host1",
   "he_mount_options": "backup-volfile-servers=host2:host3",
-  "he_bridge_if": "bridge name"
+  "he_bridge_if": "bridge name",
```

Comment:
can we change the description from "bridge name" to "interface name for bridge creation". I think the bridge name is always ovirtmgmt?